packages/react/src/components/pagination/Pagination.stories.tsx

```diff
@@ -0,0 +1,181 @@
+
+  return (
+    <>
+      <Pagination
+        language="en"
+        onChange={(event, index) => {
+          event.preventDefault();
+          setPageIndexPagination1(index);
+        }}
+        pageCount={68}
+        pageHref={() => '#'}
+        pageIndex={pageIndexPagination1}
+        paginationAriaLabel="Pagination 1"
+        siblingCount={0}
+      />
+      <Pagination
+        language="en"
+        onChange={(event, index) => {
+          event.preventDefault();
+          setPageIndexPagination2(index);
+        }}
+        pageCount={68}
+        pageHref={() => '#'}
+        pageIndex={pageIndexPagination2}
+        paginationAriaLabel="Pagination 2"
+      />
+      <Pagination
+        language="en"
+        onChange={(event, index) => {
+          event.preventDefault();
+          setPageIndexPagination3(index);
+        }}
+        pageCount={68}
+        pageHref={() => '#'}
+        pageIndex={pageIndexPagination3}
+        paginationAriaLabel="Pagination 3"
+        siblingCount={2}
+      />
+    </>
+  );
+};
+
+WithTruncation.storyName = 'With truncation';
+
+// args is required for docs tab to show source code
+// eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
+export const WithoutPrevAndNextButtons = (args) => {
+  const [pageIndex, setPageIndex] = useState<number>(7);
+
+  return (
+    <Pagination
+      hideNextButton
+      hidePrevButton
+      language="en"
+      onChange={(event, index) => {
+        event.preventDefault();
+        setPageIndex(index);
+      }}
+      pageCount={68}
+      pageHref={() => '#'}
+      pageIndex={pageIndex}
+      paginationAriaLabel="Pagination"
+    />
+  );
+};
+
+WithoutPrevAndNextButtons.storyName = 'Without prev and next buttons';
+
+// args is required for docs tab to show source code
+// eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
+export const CustomTheme = (args) => {
+  const theme = {
+    '--active-page-background-color': 'var(--color-bus)',
+  };
+
+  const [pageIndex, setPageIndex] = useState<number>(0);
+
+  return (
+    <Pagination
+      language="en"
+      onChange={(event, index) => {
+        event.preventDefault();
+        setPageIndex(index);
+      }}
+      pageCount={5}
+      pageHref={() => '#'}
+      pageIndex={pageIndex}
+      paginationAriaLabel="Pagination"
+      theme={theme}
+    />
+  );
+};
+
+CustomTheme.storyName = 'Custom theme';
+
+export const Playground = (args) => {
+  const [pageIndex, setPageIndex] = useState<number>(7);
+
+  return (
+    <Pagination
+      hideNextButton={args.hideNextButton}
+      hidePrevButton={args.hidePrevButton}
+      language={args.language}
+      onChange={(event, index) => {
+        event.preventDefault();
+        setPageIndex(index);
+      }}
+      pageCount={args.pageCount}
+      pageHref={() => '#'}
+      pageIndex={pageIndex}
+      paginationAriaLabel={args.paginationAriaLabel}
+      siblingCount={args.siblingCount}
```

Comment:
Would it be possible to add theme also here? So it could be tested in the Playground

Reply:
Good idea:

- [ ] Add theme to playground story